src/functions/meetup-callback.js

```diff
@@ -1,39 +1,24 @@
 /* eslint-disable no-console */
 const URLSearchParams = require('url').URLSearchParams
 const Got = require('got')
+const { every: Every } = require('lodash')
```

Comment:
```suggestion
const Every = require('lodash.every')
```

before: 70K
after: 13K :)